src/components/FlatList/MVCPFlatList.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable es/no-optional-chaining, es/no-nullish-coalescing-operators, react/prop-types */
```

Comment:
Note for posterity: these were disabled to keep this code as similar as possible to the upstream PR it's based on: https://github.com/necolas/react-native-web/pull/2588

Hopefully this component only lives in our codebase relatively temporarily. We probably will want to switch to FlashList soon too.